UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/Filters/BespokeExchangeSetAuthorizationFilterAttribute.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace UKHO.ExchangeSetService.API.Filters
+{
+    /// <summary>
+    /// Authorization to allow only UKHO people to create unencrypted exchange set.
+    /// </summary>
+    public class BespokeExchangeSetAuthorizationFilterAttribute : ActionFilterAttribute
+    {
+        private readonly IConfiguration configuration;
+        private const string TokenAudience = "aud";
```

Comment:
As above please change private variables to be camel case

Reply:
As discussed, private variables will use Pascal case.